.gitignore

```diff
@@ -106,6 +106,7 @@ tests-legacy/resources/modules/ps_sharebuttons
 # Installed with test data maybe via API push
 tests-legacy/resources/modules/ps_reminder
 
+tests/Resources/img/*
```

Comment:
I'm not sure we should ignore this whole folder but the internal ones instead
And if we do we should at least integrate a .gitkeep file

Did you add this for the manufacturer images created during tests? `tests/Resources/img/m/*` ?

Reply:
Files and directories are automatically created :thinking:

Reply:
empty directories should be created with .gitkeep inside (at least I remember there was some very annoying errors when image directories were missing).

Reply:
@zuk3975 Only when the creation is not automatic.